src/accountant/core.cljs

```diff
@@ -125,4 +134,6 @@
   (let [path (-> js/window .-location .-pathname)
         query (-> js/window .-location .-search)
         hash (-> js/window .-location .-hash)]
-    (secretary/dispatch! (str path query hash))))
+    (if nav-handler
```

Comment:
If we're going to put this here, should we have similar error handling in `navigate!`?